changelog/@unreleased/pr-1372.v2.yml

```diff
@@ -0,0 +1,17 @@
+type: improvement
+improvement:
+  description: |-
+    Jersey has been upgraded from 2.25.1 to 2.29. This changes a lot of transitive dependencies. In order to pass `checkRuntimeClasspathClassUniqueness`, you may need to add the following:
+    ```gradle
+    allprojects {
+      dependencies {
+        modules {
+          module('javax.ws.rs:javax.ws.rs-api') {
+              replacedBy('jakarta.ws.rs:jakarta.ws.rs-api', 'Oracle have trademarks forced Eclipse to rename https://waynebeaton.wordpress.com/2019/04/04/renaming-java-ee-specifications-for-jakarta-ee/')
+          }
+        }
+      }
+    }
+    ```
```

Comment:
This is another snippet that I would really prefer not to proliferate into thousands of gradle projects internally. Most people don't actually have `com.palantir.baseline-class-uniqueness` running, so they probably won't notice, but it's a little bit spicy anyway.